js/deer-record.js

```diff
@@ -213,11 +216,6 @@ export default class DeerReport {
                     UTILS.broadcast(undefined, DEER.EVENTS.LOADED, elem, obj)
                 }).bind(this))
                 .then(() => elem.click())
-        } else {
```

Comment:
Yes, comment on line:77. When there is no `id` all the fields are made dirty. This code isn't exactly right, but if you delete it all than a virgin form will not save fields that have not been entered. Is that the desired behavior?

```
form.deer                         <- type = Event
    input.deer.text             <- becomes dirty when user types "birthday"
    input.deer.hidden        <- is set to dirty when form loads
    input.multiselect          <- updates the hidden, but is not a deer input
    input.deer.datepicker   <- even if it has a default value, will not annotate unless input event
```

This form, if I just load the page and hit submit, saves a new Event with whatever default the hidden is and nothing else. If I add a label and hit submit, I get the same hidden value and a label, but no date, even if it has been set to a default (unless I programmatically make it dirty).

Reply:
Still needed to call this scenario out, and did so.